high_card.rb

```diff
@@ -3,3 +3,62 @@
 suits = [ "hearts", "spades", "clubs", "diamonds" ]
 
 players = []
+=begin
+1. Build a deck of shuffled cards.
+	* The deck should be an array of cards.
+	* Each card should be a mini-array with a value and suit. Example: `[8, "hearts"]`
+	* Shuffle the deck. It would look something like this...
+	```ruby
+	deck = [
+		[8, "hearts"],
+		[2, "diamonds"],
+		["J", "clubs"],
+		...
+	]
+	```
+
+deck = []
+
+ranks.each do |rank|
+  suits.each do |suit|
+    card = "#{rank} #{suit.to_s}"
+    ary = Array.new
+    Array.new(card)
```

Comment:
this is something you should be careful of. Basically each time it loops through it will create a new array and override the existing array with cards. Instead try to instantiate the array outside of the loop, and only push to that array in the each methods.